onnxruntime/core/providers/webgpu/webgpu_context.h

```diff
@@ -81,6 +81,7 @@ class WebGpuContext final {
   const wgpu::AdapterInfo& AdapterInfo() const { return adapter_info_; }
   const wgpu::Limits& DeviceLimits() const { return device_limits_; }
   bool DeviceHasFeature(wgpu::FeatureName feature) const { return device_features_.find(feature) != device_features_.end(); }
+  const wgpu::AdapterPropertiesSubgroupMatrixConfigs& SubgroupMatrixConfigs() const { return subgroup_matrix_configs_; }
```

Comment:
when building wasm target, bots report below error,
webgpu_context.h:84:15: error: no type named 'AdapterPropertiesSubgroupMatrixConfigs' in namespace 'wgpu'; did you mean 'AdapterPropertiesSubgroups'?
@fs-eire Does building wasm use old dawn files? Do you know the root-cause?

Reply:
I don't think `AdapterPropertiesSubgroupMatrixConfigs` (or the equivalent) exists in web, so Dawn does not generate the corresponding types and it makes sense to me. should remove them in WebAssembly build